bigquery_etl/cli/generate.py

```diff
@@ -19,6 +19,9 @@ def generate_group():
     generator_path = ROOT / SQL_GENERATORS_DIR
 
     for path in generator_path.iterdir():
+        if "__pycache__" in path.parts:
```

Comment:
Because we now have an init file in `sql_generators`, we _also_ get a `__pycache__` dir which has to be ignored